.github/CODEOWNERS

```diff
@@ -323,6 +323,9 @@
 # PRLabel: %Communication - Rooms
 /sdk/communication/Azure.Communication.Rooms/                      @minnieliu @paolamvhz @alexokun @Mrayyan @shwali-msft @allchiang-msft @mikehang-msft
 
+# PRLabel: %Communication - Email
```

Comment:
- `kagbakpem` does not have the required public membership to the "Azure" organization.

- `apattath` does not have the required public memberships to the "Azure" and "Microsoft" organization.